packages/react-query/src/BlockAuthors.tsx

```diff
@@ -38,20 +43,30 @@ function BlockAuthorsBase ({ children }: Props): React.ReactElement<Props> {
   useEffect((): void => {
     // No unsub, global context - destroyed on app close
     api.isReady.then((): void => {
-      let lastHeaders: HeaderExtended[] = [];
+      let lastHeaders: HeaderExtendedWithMapping[] = [];
       let lastBlockAuthors: string[] = [];
       let lastBlockNumber = '';
+      const isAuthorMapping: boolean = typeof (api.query.authorMapping?.mapping) === 'function';
```

Comment:
```suggestion
      const isAuthorMapping = isFunction(api.query.authorMapping?.mapping);
```

Reply:
I get `ReferenceError: isFunction is not defined
    at eval (webpack://@polkadot/apps/../react-query/src/BlockAuthors.tsx?:54)`

Reply:
Ahh, see the comment, basically missing the import (which I couldn't propose) - `import { formatNumber, isFunction } from '@polkadot/util';` (line 10, just above)